### DISCUSSION TITLE
Nexus OSS 3 Support

### DISCUSSION THREAD

**Discussion by AlanLg** [jreleaser/jreleaser#1452] [General]
Hello, I was wondering if Jreleaser had support for Sonatype Nexus Repository ManagerOSS 3.50.0-01

**Comment by aalmiray** [comment]
No, not at the moment. We only have support for Nexus2. Feel free to open a feature request ticket. If you happen to know where the docs for Nexus3 API may be found please link them as well. Thank you! 😅

**Comment by lbruun** [comment]
See https://github.com/jreleaser/jreleaser/issues/1836
